src/services/infra/InfraService.ts

```diff
@@ -324,32 +340,104 @@ export default class InfraService {
-      return err(error)
+      logger.error(`Failed to create '${repoName}' site on Isomer: ${error}`)
+      // requester email is guaranteed to exist as currently these are Isomer users
+      this.sendRetryToIsomerAdmin(requestor.email!, repoName)
+
+      throw error
     }
-    return ok(newLaunchParams)
+  }
+
+  sendRetryToIsomerAdmin = async (email: string, repoName: string) => {
+    const subject = `[Isomer] Failure to create domain association for ${repoName}`
+    const body = `<p>Unable to trigger create domain association for ${repoName}.</P
+    <p>If domain association was already created, please log into the amplify console and trigger a retry. </p>
+    <p>Else, resubmit the form and try again.</p>`
+    await mailer.sendMail(email, subject, body)
   }
 
   siteUpdate = async () => {
-    try {
-      const messages = await this.queueService.pollMessages()
-      await Promise.all(
-        messages.map(async (message) => {
-          const site = await this.sitesService.getBySiteName(message.repoName)
-          if (site) {
-            const updateSuccessSiteLaunchParams = {
+    const messages = await this.queueService.pollMessages()
+    await Promise.all(
+      messages.map(async (message) => {
+        const site = await this.sitesService.getBySiteName(message.repoName)
+        if (site) {
+          let updateSuccessSiteLaunchParams = {
+            id: site.id,
+            siteStatus: SiteStatus.Launched,
+            jobStatus: JobStatus.Running,
+          }
+
+          const successEmailDetails = {
+            subject: `Launch site ${message.repoName} SUCCESS`,
+            body: `<p>Isomer site ${message.repoName} was launched successfully.</p>
+            <p>You may now visit your live website. <a href="${message.primaryDomainSource}">${message.primaryDomainSource}</a> should be accessible within a few minutes.</p>
+            <p>This email was sent from the Isomer CMS backend.</p>`,
+          }
+
+          const failureEmailDetails = {
+            subject: `Launch site ${message.repoName} FAILURE`,
+            body: `<p>Isomer site ${message.repoName} was not launched successfully.</p>
+            <p>Error: ${message.statusMetadata}</p>
+            <p>This email was sent from the Isomer CMS backend.</p>
+            `,
+          }
+
+          let emailDetails: { subject: string; body: string }
+          if (message.status) {
+            emailDetails = successEmailDetails
+          } else {
+            updateSuccessSiteLaunchParams = {
+              id: site.id,
+              siteStatus: SiteStatus.Initialized,
+              jobStatus: JobStatus.Failed,
+            }
+            emailDetails = failureEmailDetails
+          }
+          await this.sitesService.update(updateSuccessSiteLaunchParams)
+          await mailer.sendMail(
+            message.requestorEmail,
+            emailDetails.subject,
+            emailDetails.body
+          )
+
+          let params
```

Comment:
this actually doesn't seem to be used at all

Reply:
removed!